src/views/NotificationsView.vue

```diff
@@ -0,0 +1,40 @@
+<template>
+  <!--wrapper div-->
+  <div class="w-screen h-screen p-20">
+    <div class="h-full">
+
+      <!--notifications-->
+      <div  class="md:pl-8 md:pr-8 2xl:pr-28 2xl:pl-28 w-full overflow-hidden">
+        <div class="">
+          <h1 class="text-4xl text-primary-orange font-bold mt-20 mb-4">
+            Notifications
+          </h1>
+        </div >
+        <div v-for="(n, i) in notificationsStore.notifications" >
+          <notification @remove='remove(i)' @redirect="redirect()" :the-notification="n" :key="n.data" ></notification> <!--:key="n.data"-->
```

Comment:
something here is wrong, I've noticed that when I get a notification and then get a new one the old one disappears - there is always only one notification there, even if I don't touch them (don't delete, and don't go to the post)